osu.Game.Rulesets.Osu/Replays/OsuFramedReplayInputHandler.cs

```diff
@@ -26,5 +26,12 @@ protected override void CollectReplayInputs(List<IInput> inputs)
             inputs.Add(new MousePositionAbsoluteInput { Position = GamefieldToScreenSpace(position) });
             inputs.Add(new ReplayState<OsuAction> { PressedActions = CurrentFrame?.Actions ?? new List<OsuAction>() });
         }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+
+            GamefieldToScreenSpace = null;
```

Comment:
rather random change? is not explained in commit message or anything

Reply:
Can probably be removed. Initially I was attempting to resolve the realm held paths, which this is one of.

Reply:
I've force pushed this out of the diff.